frontend/src/components/Toggle/index.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+
+
+class Toggle extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = { isToggleOn: true };
+
+        // This binding is necessary to make `this` work in the callback
+        this.handleClick = this.handleClick.bind(this);
+    }
+
+    handleClick() {
+        this.setState(state => ({
+            isToggleOn: !state.isToggleOn
+        }));
+    }
+
+    render() {
+        return (
+            <div style={{
+                backgroundColor: 'black', display: 'flex', justifyContent: 'center', alignItems: 'center'
+            }}>
+                <button onClick={this.handleClick}>
+                    {this.state.isToggleOn ? 'ON' : 'OFF'}
+                </button>
+            </div>
+        );
+    }
+}
+
+export default Toggle
```

Comment:
This could be done more concise with hooks and functions. Checkout @DManuel010 's `DynamicDiv` components and try to rewrite this

Reply:
fixed in new commits.